package.json

```diff
@@ -38,13 +38,15 @@
     "build:watch": "tsc -w",
     "prebuild": "yarn run clean",
     "build": "yarn run copy-templates && tsc --pretty",
-    "prepare": "node scripts/husky-install && cd frontend && yarn && cd .. && yarn run build",
+    "build:frontend": "yarn --cwd ./frontend",
+    "prepare": "node scripts/husky-install && yarn run build",
```

Comment:
Would be nice to somehow double-check if we really need this. Also, do we need to explicitly `build` here? Would be great if we could get rid of the `prepare` script altogether.